DashAISoftware/DashAI
# Model migrations with Alembic

# Summary


Adds Alembic to use for migrations and modifies dataset model deleting task_name.

## Type of change



- Back end new feature.
- Documentation.

## Changes


- Install Alembic for migration management.
- Adds How to make migrations with Alembic to the README file
- Remove task_name from dataset model with migrations
- Modify tests and validations for dataset model.

## How to Test



- Run ```pip install -r requirements.txt```
- Run ```alembic upgrade head``` to apply migrations scripts
- Run ```alembic current``` to check the current migration status
